node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
Another thing which we should keep in mind for this approach is that task can directly load a file inside module rather than loading the complete module. i.e. require("nuget-task-common/Authentication") instead of require("nuget-task-common"). Infact this is the way most of the tasks are importing modules currently. What this means is that module author will have to call setResourcePath in each of the files in module.